packages/web/src/scss/components/RadioField/_RadioField.scss

```diff
@@ -0,0 +1,75 @@
+@use 'theme';
+@use '../../tools/accessibility';
+@use '../../tools/typography';
+
+.RadioField {
+    display: inline-flex;
+    margin-block: theme.$spacing;
+}
+
+.RadioField__input {
+    display: grid;
+    width: theme.$input-width;
+    height: theme.$input-width;
+    margin: theme.$input-margin;
+    font: inherit;
+    color: theme.$input-color-unchecked;
+    background-color: transparent;
+    border: theme.$input-outline-width solid theme.$input-color-unchecked;
+    border-radius: theme.$input-border-radius;
+    appearance: none;
+    transform: translateY(-0.075em);
+    place-content: center;
+
+    &:checked {
+        border-color: theme.$input-color-checked;
+    }
+
+    &:focus-visible {
+        outline: 0;
+        box-shadow: theme.$input-focus-shadow;
+    }
+
+    &::before {
+        content: '';
+        width: theme.$input-inner-dot-width;
+        height: theme.$input-inner-dot-width;
+        border-radius: theme.$input-border-radius;
+        transform: scale(0);
+        transition: theme.$input-transition;
+        box-shadow: theme.$input-box-shadow-before;
+    }
+
+    &:checked::before {
+        transform: scale(1);
+    }
+}
+
+.RadioField__label {
+    display: inline-block;
+    margin-inline: theme.$label-margin;
```

Comment:
I think just a `margin-left` will do, that's how it is in the design too…

Reply:
margin-left is not ok if you use two fields next to each other

Reply:
You're right but that's the case of all our form fields… They need to be placed into a layout first (yes, we don't have such layout for inline forms yet 🙂). Therefore I'd incline to have it the same in all form fields so they behave the same way, now and later in a layout.